File: testbed/dotnet/SanitizerResult.cs
namespace mxssy;

public class SanitizerResult
{
    public SanitizerResult(string clean, string? serialized)
    {
        this.Clean = clean;
        this.Serialized = serialized;
    }

    public string Clean { get; }
    public string? Serialized { get; }
}

File: testbed/php/SanitizerResult.php
<?php

namespace sanitizer;

class SanitizerResult
{
    public function __construct(protected string $clean, protected ?string $serialized) {
    }

    public function get_clean(): string {
        return $this->clean;
    }

    public function get_serialized(): ?string {
        return $this->serialized;
    }
}